fly.toml

```diff
@@ -1,7 +1,5 @@
 app = "security-vulnerability-examples-next-js-postgres"
-kill_signal = "SIGINT"
-kill_timeout = 5
-processes = []
```

Comment:
new Fly.io v2 architecture for Next.js project have this configuration automatically